go.mod

```diff
@@ -4,24 +4,22 @@ go 1.13
 
 require (
 	github.com/cheggaaa/pb v1.0.28
-	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/dustin/go-humanize v1.0.0
 	github.com/fatih/color v1.7.0
 	github.com/inconshreveable/go-update v0.0.0-20160112193335-8152e7eb6ccf
 	github.com/mattn/go-colorable v0.1.1
 	github.com/mattn/go-isatty v0.0.7
 	github.com/minio/cli v1.21.0
-	github.com/minio/minio v0.0.0-20190903181048-8a71b0ec5a72
-	github.com/minio/minio-go/v6 v6.0.36
-	github.com/minio/sha256-simd v0.1.0
+	github.com/minio/minio v0.0.0-20191001201215-ff5bf519522f
+	github.com/minio/minio-go/v6 v6.0.37
```

Comment:
We can make a new release and then update this helpful for other bugs we saw in MinIO and mc @kannappanr ??